Cargo.toml

```diff
@@ -10,6 +10,6 @@ keywords = ["database", "pool"]
 edition = "2018"
 
 [dependencies]
-log = "0.4"
-parking_lot = "0.12"
-scheduled-thread-pool = "0.2"
+log = "^0.4"
+parking_lot = "^0.12"
+scheduled-thread-pool = "^0.2"
```

Comment:
Cargo by default [treats all dependencies as caret dependencies](https://doc.rust-lang.org/cargo/reference/specifying-dependencies.html#caret-requirements), so this does not change anything

Reply:
It doesn't change anything, so why is it part of the PR?

Reply:
removed, thanks for letting me know